config/prod/PsychENCODE-Globus-S3.yaml

```diff
@@ -15,10 +15,11 @@ parameters:
   # (Optional) true for read-write bucket, false (default) for read-only bucket
   AllowWriteBucket: 'true'
   # (Optional) Synapse username (default: ""), required if AllowWriteBucket=true
-  sceptre_user_data:
+sceptre_user_data:
   SynapseIDs:
     - "3434599"
```

Comment:
GrantAccess goes with parameter, not sceptre_user_data.  Please move this section to the bottom of the file.